### PR TITLE
WRN-3811: Migrate LabeledIconButton, Switch and SwitchItem unit tests to Testing Library

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated unit tests from Enzyme to Testing Library for LabeledIconButton, Switch and SwitchItem components.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-3811

### CODE REVIEW

File: SwitchItem/tests/SwitchItem-specs.js [hunk 1]
```diff
@@ -1,51 +1,46 @@
...
+		const actual = screen.getByRole('checkbox').children[2].children[0];
 
-		const expected = 1;
-		const actual = switchItem.find('Switch').length;
-
-		expect(actual).toBe(expected);
+		expect(actual).toHaveClass(expected);
 	});
 
-	test('should pass selected to Switch element', () => {
```
Comment (daniel-stoian-lgp) [inline SwitchItem/tests/SwitchItem-specs.js:20]: is there any reason why this test was deleted?

Comment (stanca-pop-lgp) [reply SwitchItem/tests/SwitchItem-specs.js:20]: No, just by mistake.

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Pull request merged**